instance/models/mixins/utilities.py

```diff
@@ -524,8 +513,10 @@ def send_acknowledgement_email_on_deployment_success(sender, **kwargs) -> None:
         return
 
     try:
-        previous_appserver = appserver.get_previous_by_created()
-    except ObjectDoesNotExist:  # Not using the strict exception to not cause circular dependencies
+        # Since the app servers are in reverse order, the element at index 1 will be
+        # the previous app server
+        previous_appserver = instance.appserver_set.all()[1]
+    except IndexError:  # Not using the strict exception to not cause circular dependencies
```

Comment:
The smallest of nits: This comment isn't needed anymore

Reply:
Removed 👍  Thanks for noticing!